core/src/test/java/io/questdb/test/std/datetime/millitime/TimeZoneRulesMillisTest.java

```diff
@@ -144,4 +145,50 @@ public void testSingle() {
             throw e;
         }
     }
+
+    @Test
+    public void testToUtcCompatibility() {
+        Set<String> allZones = ZoneId.getAvailableZoneIds();
+        List<String> zoneList = new ArrayList<>(allZones);
+        Collections.sort(zoneList);
+        List<ZoneId> zones = new ArrayList<>(zoneList.size());
+        List<TimeZoneRulesMillis> zoneRules = new ArrayList<>(zoneList.size());
+
+        for (String z : zoneList) {
+            ZoneId zone = ZoneId.of(z);
+            zones.add(zone);
+            zoneRules.add(new TimeZoneRulesMillis(zone.getRules()));
+        }
+
+        DateTimeFormatter localDateTimeFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'.000Z'");
+        long millis = Dates.toMillis(1900, 1, 1, 0, 0);
+        long epochDeadline = Dates.toMillis(2115, 12, 31, 0, 0);
```

Comment:
Please make ranges random so that test runs faster than 5s.

Reply:
Done in ec7fc3d4cd9fae05630aa2a75faccfe570a37821